code/index.html

```diff
@@ -1,30 +1,152 @@
+      <h1>Lets shape the future together!</h1>
+      <p>Become a volunteer today</p>
+      <button class="btn">Learn more</button>
+    </div>
+  </header>
+
+  <!-- Signup form -->
+  <div class="info-section-wrapper">
+    <form id="form" class="wrapper-container" action="http://httpbin.org/anything" method="POST" autocomplete="on">
+
+      <!-- first three inputs -->
+      <section class="inputs-wrapper">
+        <label class="child-form-container" for="name">
+          <input class="input custom-input" id="name" type="text" name="name" placeholder="Name" value=""
+            pattern="[a-zA-Z ]{1-32}" oninvalid="setCustomValidity('Please enter alphabets only.')" required>
+          <div class="tip" id="tip">&#128161 Example: Jane Ostin.</div>
+        </label>
+
+        <label class="child-form-container" for="email">
+          <input class="input custom-input" id="email" type="email" name="email" placeholder="Email" value=""
+            oninvalid="setCustomValidity('Please enter valid email.')" required>
+          <div class="tip" id="tip">&#128161 Example: your_email@gmail.com</div>
+        </label>
+
+        <label class="child-form-container" for="phone">
+          <input class="input custom-input" id="phone" type="tel" minlength="10" maxlength="14" name="phone"
+            pattern="^(?:0|\(?\+33\)?\s?|0033\s?)[1-79](?:[\.\-\s]?\d\d{4}$"
+            oninvalid="setCustomValidity('Please enter valid mobile number.')" placeholder="Phone Number" value=""
+            required>
+          <div class="tip" id="tip">&#128161 Should contain at least 10 digit.</div>
+        </label>
+      </section>
+
+      <!-- radio buttons set -->
+      <section class="child-form-container">
+        <p>Please specify your location:</p>
+        <div class="radio-wrapper">
+          <label class="radio-style" for="europe">
+            <input type="radio" id="europe" name="radio" value="europe">Europe
+            <span class="mark"></span></label>
+
+          <label class="radio-style" for="usa">
+            <input type="radio" id="usa" name="radio" value="usa">USA
+            <span class="mark"></span></label>
+
+          <label class="radio-style" for="australia">
+            <input type="radio" id="australia" name="radio" value="australia">Australia
+            <span class="mark"></span></label>
+
+          <label class="radio-style" for="other">
+            <input type="radio" id="other" name="radio" value="other" checked>Other
+            <span class="mark"></span></label>
+        </div>
+      </section>
+
+      <!-- checkbox -->
+      <section class="child-form-container">
+        <label class="checkbox-container" for="checkbox">
+          <div class="checkbox"> <input id="checkbox" type="checkbox" name="checkbox" value="on" required checked>
+            <span class="checkmark"></span>
+          </div>
+          <div class="terms-message"><a href="#">I argee to the Terms & Conditions.</a></div>
```

Comment:
Nice link to Terms, very professional.

Reply:
Thanks ) just looked few real webpages before starting and git inspired